src/lv_message_efficient.cc

```diff
@@ -230,7 +230,8 @@ namespace grpc_labview
         protobuf_ptr = ctx->ParseMessage(nestedMessage.get(), protobuf_ptr);
         auto nestedClusterPtr = _LVClusterHandle + fieldInfo.clusterOffset;
         auto nestedMessageValue = std::make_shared<LVNestedMessageMessageValue>(index, nestedMessage);
-        wellknown::Double2DArray::CopyToCluster(fieldInfo, nestedClusterPtr, nestedMessageValue);
+        auto constNestedMessageValue = std::static_pointer_cast<const LVMessageValue>(nestedMessageValue);
```

Comment:
This is the same thing as in cluster_copier.cc: `shared_ptr<LVNestedMessageMessageValue>` should be implicitly convertible to `shared_ptr<const LVMessageValue>`, but because the function expects a mutable ref you have to declare a separate local variable on the stack.

If `wellknown::Double2DArray::CopyToCluster` takes a `shared_ptr<const LVMessageValue>` by value or `const shared_ptr<const LVMessageValue>&` by const reference then conversions will be allowed and you won't need to declare a local variable like this.

Reply:
Updated to `const shared_ptr<const LVMessageValue>&`.